src/_pytest/python.py

```diff
@@ -753,14 +761,14 @@ def xunit_setup_method_fixture(self, request):
 
 
 class Instance(PyCollector):
-    _ALLOW_MARKERS = False  # hack, destroy later
-    # instances share the object with their parents in a way
-    # that duplicates markers instances if not taken out
-    # can be removed at node structure reorganization time
-
     def _getobj(self):
         return self.parent.obj()
 
+    @property
+    def own_markers(self) -> List[Mark]:
+        # Do not include markers from obj, coming from Class already.
```

Comment:
I guess storing a mark directly on an instance (`instance.pytestmark = [...]`) is not supported/will be ignored?